### PR TITLE
Alcail/storybook knobs to controls

### PR DESCRIPTION
**Pull request by alcail**
# What
Replaced Knobs by Controls add-on
I updated Storybook package from 6.1 to latest (6.3.6)

# Why
as Knobs add-on is being deprecated in the new versions of Storybook, controls add-on will take over

# How Tested
ran storybook and tested controls in each story

before, with knobs:
image

after, with controls:
image

# Process & policy checklist

- [ ] I have updated the project documentation to reflect my changes if necessary.
- [ ] I have read the [CONTRIBUTING](https://github.com/Azure/communication-ui-library/blob/main/CONTRIBUTING.md) documentation.

**Is this a breaking change?**

- [x] This change causes current functionality to break.

updating Storybook package version

### CODE REVIEW

File: packages/storybook/stories/controlsUtils.ts [hunk 1]
```diff
@@ -0,0 +1,299 @@
...
+  theme: { control: 'radio', options: themeChoices, defaultValue: 'Default', name: 'Theme' },
+  token: { control: 'text', defaultValue: '', name: 'Valid token for user' },
+  typingUsers: { control: 'object', defaultValue: defaultTypingUsers, name: 'Typing users' },
+  userId: { control: 'text', defaultValue: '', name: 'User identifier for user' },
+  videoGallerylayout: { control: 'select', options: VIDEO_GALLERY_LAYOUTS, defaultValue: 'default', name: 'Layout' },
+  videoTileHeight: { control: { type: 'range', min: 300, max: 800, step: 10 }, defaultValue: 300, name: 'Height (px)' },
+  videoTileWidth: { control: { type: 'range', min: 400, max: 1200, step: 10 }, defaultValue: 400, name: 'Width (px)' }
+};
+
+export const hiddenControl = { control: false, table: { disable: true } };
```
Comment (JamesBurnside) [inline packages/storybook/stories/controlsUtils.ts:273]: 😍

File: packages/storybook/stories/MeetingComposite/__snapshots__/BasicExample.stories.storyshot [hunk 2]
```diff
@@ -0,0 +1,32 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```
Comment (JamesBurnside) [inline packages/storybook/stories/MeetingComposite/__snapshots__/BasicExample.stories.storyshot:1]: For some of these I see a new snapshot file but not a deleted one. I think there might be redundant storyshot files where story filename renames happened.
i.e. `BasicExample.tsx` became `BasicExample.stories.tsx` so jest created a new `BasicExample.stories.storyshot` but did not delete the old `BasicExample.storyshot`

**Review by JamesBurnside — APPROVED** [review]
Thanks for refactoring the controls into an exported location - so mcuh cleaner and eaier to read! 😍 Just one note there may be now-redundant storyshot files but otherwise looks great!!

File: packages/storybook/stories/MeetingComposite/__snapshots__/BasicExample.stories.storyshot (hunk 2 above)
Comment (alcail) [reply packages/storybook/stories/MeetingComposite/__snapshots__/BasicExample.stories.storyshot:1]: will check that

Comment (alcail) [reply packages/storybook/stories/MeetingComposite/__snapshots__/BasicExample.stories.storyshot:1]: storyshot files exist only for .stories files, so no redundancies but I need to remove the ones that are not stories anymore

Comment (alcail) [reply packages/storybook/stories/MeetingComposite/__snapshots__/BasicExample.stories.storyshot:1]: done :)

**Review by mgamis-msft — APPROVED** [review]

**Pull request merged**